nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestSetPropertiesAction.java

```diff
@@ -62,7 +62,16 @@
     public TransactionalFeature txFeature;
 
     @Test
-    public void testSetProperties() throws Exception {
+    public void testSetPropertiesAsAdmin() throws Exception {
+        testSetProperties("Administrator");
+    }
+
+    @Test
+    public void testSetPropertiesAsNonAdmin() throws Exception {
+        testSetProperties("Tutu");
```

Comment:
if Tutu is supposed to be a user name make it obvious and choose a common english name without uppercase char.